linux_os/guide/system/network/network-iptables/iptables_ruleset_modifications/ip6tables_rules_for_open_ports/sce/shared.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
```

Comment:
```suggestion
```

Reply:
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#tests
It is not specific for `sce` but it is reasonable to adopt here.